yarn.lock

```diff
@@ -3022,25 +3022,10 @@ acorn-walk@^6.0.1:
   resolved "https://registry.yarnpkg.com/acorn-walk/-/acorn-walk-6.2.0.tgz#123cb8f3b84c2171f1f7fb252615b1c78a6b1a8c"
   integrity sha512-7evsyfH1cLOCdAzZAd43Cic04yKydNx0cF+7tiA19p1XnLLPU4dpCQOqpjqwokFe//vS0QqfqqjCS2JkiIs0cA==
 
-acorn@5.X, acorn@^5.0.3, acorn@^5.5.3:
-  version "5.7.4"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-5.7.4.tgz#3e8d8a9947d0599a1796d10225d7432f4a4acf5e"
-  integrity sha512-1D++VG7BhrtvQpNbBzovKNc1FLGGEE/oGe7b9xJm/RFHMBeUaUGpluV9RLjZa47YFdPcDAenEYuq9pQPcMdLJg==
-
-acorn@^6.0.1:
-  version "6.2.0"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.2.0.tgz#67f0da2fc339d6cfb5d6fb244fd449f33cd8bbe3"
-  integrity sha512-8oe72N3WPMjA+2zVG71Ia0nXZ8DpQH+QyyHO+p06jT8eg8FGG3FbcUIi8KziHlAfheJQZeoqbvq1mQSQHXKYLw==
-
-acorn@^6.2.1:
-  version "6.3.0"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.3.0.tgz#0087509119ffa4fc0a0041d1e93a417e68cb856e"
-  integrity sha512-/czfa8BwS88b9gWQVhc8eknunSA2DoJpJyTQkhheIf5E48u1N0R4q/YxxsAeqRrmK9TQ/uYfgLDfZo91UlANIA==
-
-acorn@^7.1.0:
-  version "7.1.0"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-7.1.0.tgz#949d36f2c292535da602283586c2477c57eb2d6c"
-  integrity sha512-kL5CuoXA/dgxlBbVrflsflzQ3PAas7RYZB52NOm/6839iVYJgKMJ3cQJD+t2i5+qFa8h3MDpEOJiS64E8JLnSQ==
+acorn@5.X, "acorn@>=5.7.4 <6.0.0 || >=6.4.1 <7.0.0 || >=7.1.1", acorn@^5.0.3, acorn@^5.5.3, acorn@^6.0.1, acorn@^6.2.1, acorn@^7.1.0:
```

Comment:
I've looked at resolutions for these packages, but because you can't really select what package versions to set them for, this can be problematic. As you can see here, all `acorn` versions are being set to 7.1.1, even when the software depends on older 5.7.4. How can we be sure there are no breaking changes?

Reply:
Good point. I did a quick look through `master`s yarn.lock, and I see only 3 instances of `<5.7.4`: `jsdom@^11.5.1`, `@gulp-sourcemaps/identity-map@1.X`, `gulp-sourcemaps@2.6.5`. `jsdom` is used in jest dependency tree, and gulp stuff is build tools, which IMO easy to verify it's functionality.

Reply:
Also, its worth to mention that there are no breaking changes between 5.0.X and 5.7.X, since it's a server. Confirmed this by going through `acorn` release notes

Reply:
Nice! Since there's not many use cases of it, we can just specify the version for each of these, right? See my other comment on that please.